### PR TITLE
nrf_security: Move PSA_WANT symbols from Zephyr to nrf_security

### PR DESCRIPTION
**Pull request by Vge0rge**
The Nordic only PSA_WANT symbols make more sense to be placed here instead of adding a noup commit in Zephyr

### CODE REVIEW

**Review by tomi-font — APPROVED** [review]

**Comment by tejlmand** [comment]
so question goes for this commit: https://github.com/nrfconnect/sdk-zephyr/commit/0892badd136949deb79a9098c2c4c93cb0e86a3f

why are we having this as a noup instead of keeping it downstream ?

**Review by frkv — APPROVED** [review]
LGTM (minus removal of CFB and OFB which we don't want to continue supporting)

File: subsys/nrf_security/Kconfig
Comment (tomi-font) [inline subsys/nrf_security/Kconfig:56]:
```suggestion
# Include Nordic-only PSA configurations
```

File: subsys/nrf_security/Kconfig.psa.nordic
Comment (tomi-font) [inline subsys/nrf_security/Kconfig.psa.nordic:7]:
```suggestion
comment "Nordic-added meta types"
```

**Review by frkv — APPROVED** [review]
LGTM (pending correct SHA and green CI)

**Review by tomi-font — COMMENTED** [review]
Nice little cleanup. (Two tiny remarks, do what you want with them.)
I'm thinking, regarding CFB and OFB removal, maybe this also requires some update to the following files?
- `doc/nrf/libraries/security/nrf_security/doc/driver_config.rst`
- `subsys/nrf_security/src/drivers/Kconfig.psa_accel`
- `subsys/nrf_security/include/psa/core_unsupported_ciphers_check.h`

**Comment by Vge0rge** [comment]
@nrfconnect/ncs-co-build-system @nrfconnect/ncs-code-owners @nrfconnect/ncs-aegir-doc  please review

**Review by mia-ko — CHANGES_REQUESTED** [review]
These are just removed from the rst file, so it's all good from the doc perspective, except from a missing changelog entry.

**Comment by Vge0rge** [comment]
> These are just removed from the rst file, so it's all good from the doc perspective, except from a missing changelog entry.

Added something there.

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:813]:
```suggestion
* :ref:`nrf_security` library:

  * Removed the Kconfig options ``CONFIG_PSA_WANT_ALG_CFB`` and ``CONFIG_PSA_WANT_ALG_OFB`` since the Cipher Feedback (CFB) mode and the Output Feedback (OFB) mode are not tested in the test framework.
```

**Review by mia-ko — CHANGES_REQUESTED** [review]

**Review by mia-ko — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Pull request merged**
